tests/test_ping.py

```diff
@@ -0,0 +1,20 @@
+
+
+def test_http_1_0_ping_response(client):
+    response = client.get('/ping', environ_overrides={'SERVER_PROTOCOL': 'HTTP/1.0'})
+
+    assert response.status_code == 200
+    assert response.content_type == 'text/plain; charset=utf-8'
+    assert response.data == b'pong'
```

Comment:
do you usually use `response.data` (bytes) or `response.text` (string)?

Reply:
Both can be used. I guess `response.data` will always show the response body but `response.text` will only show text. Given the response should be in plain text. I can change this to `response.text`.

Reply:
:+1: given the strict content type, I agree; would normally reserve `data` for binary responses

Reply:
updated to `response.text`.

Reply:
Reverted as `flask.Response` does not have a `text` attribute.